### ISSUE TITLE
Link to latest is wrong for VS 2022

### ISSUE THREAD

**Issue by randomascii** [chromium/vs-chromium#87]
The VsChromium main page at https://chromium.github.io/vs-chromium/#installation links to https://github.com/chromium/vs-chromium/releases/tag/v0.9.37 as the latest release. However this doesn't work with VS 2022. Finding the correct release is non-obvious. Suggestions:

Update the main page to link to both the VS 2022 and previous-versions, with VS 2022 getting more prominent billing. Or, link directly to v0.9.39 since it looks like it supports all VS versions - just take away the pre-release tag.

v0.9.38 should probably be deleted.

**Comment by cyberfoxmeow** [comment]
I also met this issue.

Is it possible to download the old VS version and use this extension?

**Comment by duaneking** [comment]
Yeah if you try to install that build it'll just give you an exception and say that it's not compatible with anything that's installed even if you have Visual Studio 2022 fully installed.

Just hit this myself and yet GitHub/etc isn't giving me a better version to install, either.

**Comment by duaneking** [comment]
Referencing #88 to help others - use the pre-release version that actually supports 2022

**Comment by randomascii** [comment]
Reiterating the last comment, this is the installer that should be used for VS 2022. I have been using it for three years without problems:

https://github.com/chromium/vs-chromium/releases/tag/v0.9.39

It is too bad that it is still tagged as pre-release.

**Comment by randomascii** [comment]
Ping! I know how to find the latest version for VS 2022 but most people don't. It would be great to make v0.9.39 the official release. It's been working well for me for years
